docs/docs/api/09-Remote-Device-Controller.md

```diff
@@ -0,0 +1,114 @@
+# Remote Device Controller
+
+:::caution
+
+The remote device controller is currently in __preview__ release and only works on __Windows__ and for __Android__ devices controlled from a __Windows__ machine.
+
+The installer only works on __Windows__.
+
+:::
+
+## Introduction & Setup
+
+### Installer
+
+1. Download the installer from [here](https://askui-public.s3.eu-central-1.amazonaws.com/releases/preview/v23.10.01/askui+Installer.exe).
```

Comment:
Can't we go over Cloudfront? Didn't you set up a Cloudfront distribution? Because downloading it from here directly is costly and slow.

Reply:
Yes. Already forgot about that as I set it up before my vacation.